library/kotlin/src/io/envoyproxy/envoymobile/Counter.kt

```diff
@@ -0,0 +1,24 @@
+package io.envoyproxy.envoymobile
+
+import io.envoyproxy.envoymobile.engine.EnvoyEngine
+import java.lang.ref.WeakReference
+
+/**
+ * A stat.
+ *
+ * Current the supported stat type is counter, and it can increment.
```

Comment:
I assume we will create different types for non-Counter stats, so maybe this docstring can be updated to simply describe the Counter?